src/bindings/python/src/pyopenvino/core/core.cpp

```diff
@@ -544,19 +545,44 @@ void regclass_Core(py::module m) {
            const py::object& model_stream,
            const std::string& device_name,
            const std::map<std::string, py::object>& properties) {
-            auto _properties = Common::utils::properties_to_any_map(properties);
+            const auto _properties = Common::utils::properties_to_any_map(properties);
             if (!(py::isinstance(model_stream, pybind11::module::import("io").attr("BytesIO")))) {
                 throw py::type_error("CompiledModel.import_model(model_stream) incompatible function argument: "
                                      "`model_stream` must be an io.BytesIO object but " +
                                      (std::string)(py::repr(model_stream)) + "` provided");
             }
             model_stream.attr("seek")(0);  // Always rewind stream!
-            std::stringstream _stream;
-            _stream << model_stream
-                           .attr("read")()  // alternative: model_stream.attr("get_value")()
-                           .cast<std::string>();
-            py::gil_scoped_release release;
-            return self.import_model(_stream, device_name, _properties);
+            const std::string filename = "model_stream.txt";
```

Comment:
Why use hardcoded name.
Is there any risk of collision to access same file by many requests. 
Consider use unit test approach where temporary file has some random sufix/prefix?

Reply:
Added random suffix.